AK/BuiltinWrappers.h

```diff
@@ -134,7 +134,7 @@ inline constexpr int count_leading_zeroes_safe(IntType value)
 template<Integral IntType>
 inline constexpr int bit_scan_forward(IntType value)
 {
-#if defined(AK_COMPILER_CLANG) || defined(AK_COMPILER_GCC)
+#if defined(AK_COMPILER_CLANG) || (defined(AK_COMPILER_GCC) && (!defined(__riscv) || defined(__riscv_zbb)))
```

Comment:
Can we use ARCH(RISCV) rather than checking the actual `__riscv` macro? Is `zbb` a common extension for "desktop-class" RISC-V boards? I'm guessing you added this b/c your dev kit doesn't have it?